pkgs/development/python-modules/apptools/default.nix

```diff
@@ -17,7 +17,6 @@ buildPythonPackage rec {
     configobj
     six
     traitsui
-  ] ++ lib.optionals (pythonOlder "3.9") [
```

Comment:
Without this change I got 
```
python3.9-apptools> ____ ERROR collecting apptools/preferences/tests/test_scoped_preferences.py ____
python3.9-apptools> ImportError while importing test module '/build/apptools-5.1.0/apptools/preferences/tests/test_scoped_preferences.py'.
python3.9-apptools> Hint: make sure your test modules/packages have valid Python names.
python3.9-apptools> Traceback:
python3.9-apptools> /nix/store/7hz3pcgi8pm3n6bn2fk2w32kd4nvf8g6-python3-3.9.2/lib/python3.9/importlib/__init__.py:127: in import_module
python3.9-apptools> return _bootstrap._gcd_import(name[level:], package, level)
python3.9-apptools> apptools/preferences/tests/test_scoped_preferences.py:19: in <module>
python3.9-apptools> from importlib_resources import files
python3.9-apptools> E   ModuleNotFoundError: No module named 'importlib_resources'
```

Reply:
importlib_resources is the backport of importlib.resources. If importlib.resources fails to import you should try to import the backport importlib_resources.

Reply:
That makes sense, but I don't really want to patch the apptools source to do that...

Reply:
Patching here would be the clean solution. Maybe there is already an upstream pr we can fetch? If not I think adding a comment with an upstream issue would be ideal.

Reply:
https://github.com/enthought/apptools/pull/284